### PR TITLE
Add page aliases

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-450.surge.sh

Updated pages:
- [Connecting applications](https://neo4j-docs-aura-450.surge.sh/aura/auradb/connecting-applications/overview)
- [Connecting to an instance](https://neo4j-docs-aura-450.surge.sh/aura/auradb/getting-started/connect-database)
- [Creating an instance](https://neo4j-docs-aura-450.surge.sh/aura/auradb/getting-started/create-database)
- [Querying an instance](https://neo4j-docs-aura-450.surge.sh/aura/auradb/getting-started/query-database)
- [Importing an existing database](https://neo4j-docs-aura-450.surge.sh/aura/auradb/importing/import-database)
- [Importing data](https://neo4j-docs-aura-450.surge.sh/aura/auradb/importing/importing-data)
- [Advanced metrics](https://neo4j-docs-aura-450.surge.sh/aura/auradb/managing-databases/advanced-metrics)
- [Instance actions](https://neo4j-docs-aura-450.surge.sh/aura/auradb/managing-databases/database-actions)
- [Monitoring](https://neo4j-docs-aura-450.surge.sh/aura/auradb/managing-databases/monitoring)

File: publish.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    common-license-page-uri: https://neo4j.com/docs/license/
+    # apoc procedures contain content that look like attributes, but aren't
+    params: '\{params}'
+    config: '\{config}'
+    data: '\{data}'
+    properties: '\{properties}'
+    json: '\{json}'
+    maps: '\{maps}'
+    neo4j-base-uri: ''
+    neo4j-docs-base-uri: https://neo4j.com/docs
```
Comment (recrwplay) [inline publish.yml:64]:
```suggestion
    neo4j-docs-base-uri: /docs
```

Comment (AlexicaWright) [reply publish.yml]: 🖖

**Review by recrwplay — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-450.surge.sh

Updated pages:
- [Connecting applications](https://neo4j-docs-aura-450.surge.sh/aura/auradb/connecting-applications/overview)
- [Connecting to an instance](https://neo4j-docs-aura-450.surge.sh/aura/auradb/getting-started/connect-database)
- [Creating an instance](https://neo4j-docs-aura-450.surge.sh/aura/auradb/getting-started/create-database)
- [Querying an instance](https://neo4j-docs-aura-450.surge.sh/aura/auradb/getting-started/query-database)
- [Importing an existing database](https://neo4j-docs-aura-450.surge.sh/aura/auradb/importing/import-database)
- [Importing data](https://neo4j-docs-aura-450.surge.sh/aura/auradb/importing/importing-data)
- [Advanced metrics](https://neo4j-docs-aura-450.surge.sh/aura/auradb/managing-databases/advanced-metrics)
- [Instance actions](https://neo4j-docs-aura-450.surge.sh/aura/auradb/managing-databases/database-actions)
- [Monitoring](https://neo4j-docs-aura-450.surge.sh/aura/auradb/managing-databases/monitoring)

**Pull request merged**